packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -104,6 +161,50 @@ const ProductTile = (props) => {
                     )}
                 </Box>
 
+                {/* Swatches */}
+                {variationAttributes
+                    ?.filter(({id}) => selectableAttributeId == id)
+                    ?.map(({id, values}) => {
+                        const attributeId = id
+                        return (
+                            <SwatchGroup key={id}>
+                                {values?.map(({href, name, image, value}) => {
+                                    const content = image ? (
+                                        <Box
+                                            height="100%"
+                                            width="100%"
+                                            minWidth="32px"
+                                            backgroundRepeat="no-repeat"
+                                            backgroundSize="cover"
+                                            backgroundColor={name.toLowerCase()}
```

Comment:
this assumes name is 'color', what happens if the name is different?

Reply:
I think what we should do here is select 'circle' as the only current option. I only say this because I don't want too much scope creep in making everything under the rainbow configurable. We can open it up for configurability after the initial version